NServiceBus.Transport.AzureServiceBus 3.1.0 Updates (#5923)
Particular/docs.particular.net: transports/azure-service-bus/configuration_entity-topology_asbs_[,2].partial.md

### Topology

> * `TopicName(string)`: The name of the topic used to publish events between endpoints. This topic is shared by all endpoints, so ensure all endpoints configure the same topic name. Defaults to `bundle-1`. Topic names must adhere to the limits outlined in [the Microsoft documentation on topic creation](https://docs.microsoft.com/en-us/rest/api/servicebus/create-topic).

Reviewer: ```suggestion
* `TopicName(string)`: The topic's name used to publish events between endpoints. All endpoints share this topic, so ensure all endpoints specify the same topic name. Defaults to `bundle-1`. Topic names must adhere to the limits outlined in [the Microsoft documentation on topic creation](https://docs.microsoft.com/en-us/rest/api/servicebus/create-topic).
```
